e2e/autoscript-suites/project.json

```diff
@@ -8,7 +8,7 @@
       "options": {
         "commands": [
           {
-            "command": "concurrently --kill-others --success first  \"playwright test\" \"npm run start:server\"",
+            "command": "concurrently --kill-others --success first  \"playwright test --config=./e2e/autoscript-suites/playwright.config.ts\" \"npm run start:server\"",
```

Comment:
target the specific config in the folder (root workspace was default previously).